### ISSUE TITLE
Expose Trunk

### ISSUE THREAD

**Issue by apabari** [meg768/homebridge-my-tesla#11]
Is it possible to expose the trunk as an item that can be opened or closed?

**Comment by meg768** [comment]
Yes, possible to open it, both trunk and frunk. Not sure about current state, but to open is not a problem. Will look into it. Want it myself.

**Comment by samleemc** [comment]
Hi, I think your project is most user friendly! But it will be great if we can have trunk and frunk! Please keep up the work! Looking forward to your update!

**Comment by raphaeldavid** [comment]
with confirmation.
